extension/entities/Question.manifest

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2017 Google Inc. All rights reserved.
+# This code may only be used under the BSD style license found at
+# http://polymer.github.io/LICENSE.txt
+# Code distributed by Google as part of this project is also
+# subject to an additional IP rights grant found at
+# http://polymer.github.io/PATENTS.txt
+
+
+# replace this with a straight reference to https://schema.org/Answer
+
+import 'https://polymerlabs.github.io/arcs-cdn/v0.0.4/entities/Thing.manifest'
+
+schema Question extends Thing
```

Comment:
I think it's probably simpler to put these all in the same file, unless you expect someone else to depend on them.